### DISCUSSION TITLE
Maven dependency update not generated when version number is represented by a property with additional value appended

### DISCUSSION THREAD

**Discussion by dandn** [renovatebot/renovate#18139] [Request Help] [answered]
I'm running renovate using [whitesource/renovate-on-prem](https://github.com/whitesource/renovate-on-prem/releases) v2.5.1.

I have an issue with the following configuration that is not be acted upon:
```
<properties>
        <scala.binary.version>2.13</scala.binary.version>
        <scala.version>${scala.binary.version}.8</scala.version>
</properties>

<dependencyManagement>
    <dependencies>
            <dependency>
                <groupId>org.scala-lang</groupId>
                <artifactId>scala-library</artifactId>
                <version>${scala.version}</version>
            </dependency>
    </dependencies>
</dependencyManagement>
```
As of this writing, an update to scala-library 2.13.9 should be generated, but it is not.

A workaround is to simply declare the scala version as follows:
```
 <scala.version>2.13.8</scala.version>
```
In this case a PR is generated.

Is this something that should be treated as an issue?

**Comment by viceice** [comment, marked as answer]
no, i think renovate shouldn't support garbled versions. so you should a single version property
